pkg/agent/openflow/packetin.go

```diff
@@ -78,9 +78,12 @@ const (
 
 	// We use OpenFlow Meter for packetIn rate limiting on OVS side.
 	// Meter Entry ID.
-	PacketInMeterIDNP  = 1
-	PacketInMeterIDTF  = 2
-	PacketInMeterIDDNS = 3
+	// 1-255 are reserved for Egress QoS. The Egress QoS meterID leverage the same
```

Comment:
The Egress QoS meterID uses the same value as

Reply:
Done.